modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +44,63 @@ spec:
       istiod:
         type: PrivateKey
         privateKey:
-          rootCADir:  /etc/cacerts
+          rootCADir: /etc/cacerts
 ----
 
+[NOTE]
+====
+After creating/changing/deleting the `cacert` secret, the control plane `istiod` and `gateway` pods must be restarted so the changes go into effect. Use the following command to restart the pods: `oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway)'`.
+
+Restart the application pods so that the sidecar containers pick up the changes as well.
+====
+
 [id="ossm-cert-manage-verify-cert_{context}"]
 == Verifying your certificates
 
-Use the Bookinfo sample application to verify your certificates are mounted correctly. First, retrieve the mounted certificates. Then, verify the certificates mounted on the pod.
+Use the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] to verify that the workload certificates are signed by the certificates that were plugged into the CA. This requires you have `openssl` installed on your machine
 
-. Store the pod name in the variable `RATINGSPOD`.
-+
-[source,terminal]
-----
-$ RATINGSPOD=`oc get pods -l app=ratings -o jsonpath='{.items[0].metadata.name}'`
-----
+. Extracting certificates from bookinfo workloads
 +
-. Run the following commands to retrieve the certificates mounted on the proxy.
+Wait about 1 minute for the mTLS policty to take effect before retrieving the certificate chain of bookinfo. If Bookinfo was already deployed when you created the secret above, you need to restart the pods so they pick up the changes (see note above). Replace `<bookinfo-project>` with the project name where bookinfo workload is installed.
 +
 [source,terminal]
 ----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /var/run/secrets/istio/root-cert.pem > /tmp/pod-root-cert.pem
+$ # Restart the bookinfo pods if they were already deployed. Not needed if they are deployed after the secret changes above.
```

Comment:
Did you mean for these first two lines to be commented out?  
If you did want these lines as comments, I'd move then out of the code block, so that if customers use the copy/paste function, they don't get these unneeded lines.

Reply:
Broke them out as a note.